samples/csharp_dotnetcore/80.skills-simple-bot-to-bot/SimpleRootBot/appsettings.json

```diff
@@ -8,5 +8,13 @@
       "AppId": "",
       "SkillEndpoint": "http://localhost:39783/api/messages"
     }
-  ]
+  ],
+  "runtimeSettings:skills": {
+    // This is a comma separated list with the App IDs of all skills that can callback to this root bot.
+    // This setting is used in AllowedCallersClaimsValidator.
+    // Examples: 
+    //    [ "*" ] allows all skills.
+    //    [ "AppId1", "AppId2" ] only allows access to skill bots with "AppId1" and "AppId2".
+    "AllowedCallers": [ "*" ]
+  }
```

Comment:
Is this right?! Shouldn't allowed callers be applied only to the skill bot, and not to the consumer bot?